contract.py

```diff
@@ -100,14 +100,14 @@ def decimals():
     """
     Returns the amount of decimals of the token
     """
-    return DECIMALS
+    return DECIMALS + 0
```

Comment:
Adding `0` to `DECIMALS` does nothing. `DECIMALS` is `8` and adding `0` does not change it's value

Reply:
+0 will change the data type to integer type, which is very important when calling across contracts.

Reply:
neovm does not support floating point numbers, so the number is an integer by default